inference/wasm/tests/engine/worker-global-scope-simulator.mjs

```diff
@@ -0,0 +1,157 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { isMainThread, parentPort } from "worker_threads";
```

Comment:
Suggestion: I think this can be changed to `node:worker_threads` which makes it a bit more explicit that this is a core node module.